### PR TITLE
NF: replace sting buffer by builder

### PR DESCRIPTION
**Pull request by Arthur-Milchior**
Slightly more efficient and remove a warning

### CODE REVIEW

**Review by mikehardy — APPROVED** [review]
ok

**Comment by Arthur-Milchior** [comment]
Here I tried to add a new fatal warning to avoid the same warning pop-up again ignored. Trying to follow @david-allison-1 's advice. However, it does not seems to work, because when I just change the xml and leaves the buffer, it compiles properly

File: lint-release.xml [hunk 1]
```diff
@@ -30,6 +30,7 @@
     <issue id="ViewTag" severity="fatal" />
     <issue id="WrongConstant" severity="fatal" />
     <issue id="WrongViewCast" severity="fatal" />
+    <issue id="StringBufferMayBeStringBuilder" severity="fatal"/>
```
Comment (david-allison-1) [inline lint-release.xml:33]: Seems like it's IntelliJ, rather than Android Studio, so doesn't use gradle's lint. That's a shame

```suggestion
```

Comment (Arthur-Milchior) [reply lint-release.xml]: This line is removed

**Comment by Arthur-Milchior** [comment]
So is there anything that can be done to ensure that compilation fail on those warnings ? I can't easily find it by searching online. I tried to set the warning to "error" in android studio. It does not seems to change anything

**Pull request merged**